app/views/schools/core_programme/materials/info.html.erb

```diff
@@ -1,228 +1,9 @@
 <% content_for :before_content, govuk_back_link( text: "Back", href: schools_cohort_path(@cohort.start_year)) %>
```

Comment:
Would you be able to check this group of pages for page titles please? I noticed some missing when looking at google analytics